Solutions/ZoomReports/Data Connectors/ZoomSentinelConnector/__init__.py

```diff
@@ -9,97 +8,197 @@
 import re
 import os
 import logging
+import time
 from .state_manager import StateManager
 
-jwt_api_key = os.environ['ZoomApiKey']
-jwt_api_secret = os.environ['ZoomApiSecret']
+zoom_account_id = os.environ['AccountID']
+zoom_client_id = os.environ['ClientID']
+zoom_client_secret = os.environ['ClientSecret']
 customer_id = os.environ['WorkspaceID']
 shared_key = os.environ['WorkspaceKey']
 connection_string = os.environ['AzureWebJobsStorage']
 logAnalyticsUri = os.environ.get('logAnalyticsUri')
 table_name = "Zoom"
 chunksize = 10000
+retry = 3 ## To do : need to move function app configuration
+error=False
+#Max script execution
+SCRIPT_EXECUTION_INTERVAL_MINUTES = 30
```

Comment:
Moving this to function app configuration.

Reply:
Done ,moved to function app config,working on local testing,once local testing completed with mock end point,will deploy and test the new and old version

Reply:
Deployed and tested,also tested locally with mock end point